releng/cody-update-site/site.xml

```diff
@@ -1,5 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <site>
-   <feature url="features/cody_feature_0.1.0.202407041638.jar" id="cody_feature" version="0.1.0.202407041638"/>
-   <feature url="features/cody_feature_0.2.1.202407051552.jar" id="cody_feature" version="0.2.1.202407051552"/>
+   <feature url="features/cody_feature_0.2.2.202407311435.jar" id="cody_feature" version="0.2.2.202407311435"/>
```

Comment:
In order to build it to be packaged and installed locally, I had to delete the previous two entries. Is that expected or should I revert this?

Reply:
I normally only use the debug launcher, which doesn't change this XML file. You shouldn't need to build an actual plugin to install it locally.

I do get conflicts here when I cut new releases in the `releases` git branch. I am planning to move the release process to Bazel and only support one version via Site URL (and github automation). [CODY-3197: Automated releases with Bazel](https://linear.app/sourcegraph/issue/CODY-3197/automated-releases-with-bazel)